corral/vcs/bzr.pony

```diff
@@ -1,6 +1,8 @@
 use "files"
 
 primitive BzrVcs is Vcs
+  """Placeholder for Bazaar VCS"""
```

Comment:
I think this should be 3 lines but the style guide isn't clear. @Theodus we should clarify that.

Reply:
There's precedent in other projects (like ponycheck) to keep the `"""` tokens on separate lines. I'll make a note to myself to clarify that in the guide.